### DISCUSSION TITLE
keyball61の部品を購入したいです

### DISCUSSION THREAD

**Discussion by Forseti4** [Yowkees/keyball#422] [Q&A]
こんにちは、ピンヘッダーが届いて再度はんだ付けしましたが、'DFG'が押し続けられる状況は変わらず、何度かはんだを取り除いて作業を繰り返した結果、今は反応がなくなりました.. :( 右側の基板やダイオード、ピンヘッダー、スイッチなどを購入することは可能ですか？再度購入して作業をしたいです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

Keyballキットご購入者様限定で、ミドル基板や再半田付けに必要なキーソケット、ダイオード等のセットを販売可能です。
セット内容や購入方法等、詳しくは下記の白銀ラボウェブショップのお問合せよりご連絡いただき、ご相談させてください。
https://shirogane-lab.net/inquiry

以上、お手数ですがご連絡よろしくお願いいたします。
